docs/_includes/Running-Alluxio-on-Mesos/alluxio-mesos.md

```diff
@@ -1,3 +1,3 @@
 ```bash
-$ ./integration/bin/alluxio-mesos.sh mesosMaster:5050 // address of Mesos master
+$ ./integration/mesos/bin/alluxio-mesos-start.sh mesosMaster:5050 // address of Mesos master
```

Comment:
the motivation for that change was that we also had a script for stopping mesos and the two scripts were called `alluxio-mesos.sh` and `stop-alluxio-framework.sh` ... now they are called `alluxio-mesos-start.sh` and `alluxio-mesos-stop.sh` .... for yarn, AFAIK, we do not have a script for stopping the application master ... if you would prefer, I would be happy to rename `alluxio-yarn.sh` to `alluxio-yarn-start.sh` and adding `alluxio-yarn-stop.sh` (I might need to ask @aaudiber or @apc999 for help with that)